composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "laravel/installer": "^1.3"
```

Comment:
Fair enough. 
The only thing I can think of would be supporting the --dev flag, which was added in 1.3.1 so I suppose we do need 1.3.